### DISCUSSION TITLE
Better android video player for android/wpf?

### DISCUSSION THREAD

**Discussion by vynxc** [unoplatform/uno#10142] [Q&A]
The media player element doesn't have much implementation on android. I'd like to make a ui with play, pause, next video, skip 10, back 10, and a slider that allows to change time. Currently that doesn't seem possible using the media player element.

**Comment by jeromelaban** [comment]
The MediaPlayerElement on Android supports the features you've listed. Here's [one sample we use to validate](https://github.com/unoplatform/uno/blob/master/src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/MediaPlayerElement/MediaPlayerElement_Full.xaml) some of the features of that control.

Can you explain what you're getting when using it?

**Reply by vynxc** [reply]
MediaPlayerElement current state not implemented into uno. I was trying to make a custom pause button and was using that to change the icon. I assumed the rest wasn't implemented aswell. I haven't been able to work on it much but I decided to make a discussion to scape some knowledge for when I can.

**Reply by jeromelaban** [reply]
Have you actually tried using it in an app?

**Reply by vynxc** [reply]
Yeah, didn't work. I'll have to do it another way.

**Reply by jeromelaban** [reply]
Can you explain what did not work? Can you make a screen capture?
